CONTRIBUTING.md

```diff
@@ -57,25 +57,39 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Releases
+
+Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases.
+
+- Minor versions CAN be planned and scheduled twice / year.
```

Comment:
nit: I know this might be obvious but I'm assuming that `year` is calendar year. For example, Jan 1, 2020 - Jan 1, 2021.

Reply:
Definitely can add it if you prefer, although not sure if that's a hard rule that it was to be in a calendar year